src/hotspot/share/nmt/vmtCommon.cpp

```diff
@@ -0,0 +1,402 @@
+/*
```

Comment:
Is this all of the code that is still required by the reporter to be able to function?